python/cugraph-pyg/cugraph_pyg/tests/test_cugraph_loader.py

```diff
@@ -159,33 +184,45 @@ def test_cugraph_loader_from_disk_subset():
         num_samples += 1
         assert sample["t0"]["num_nodes"] == 7
         # correct vertex order is [0, 1, 2, 6, 4, 3, 5]; x = [1, 2, 3, 7, 5, 4, 6]
-        assert sample["t0"]["x"].tolist() == [1, 2, 3, 7, 5, 4, 6]
-        assert list(sample[("t0", "knows", "t0")]["edge_index"].shape) == [2, 7]
+        assert sample["t0"]["x"].tolist() == [3, 4, 5, 6, 7, 8, 9]
+
+        edge_index = sample[("t0", "knows", "t0")]["edge_index"]
+        assert list(edge_index.shape) == [2, 8]
+
+        assert (
+            edge_index[0].tolist()
+            == bogus_samples.sources.dropna().values_host.tolist()
+        )
+        assert (
+            edge_index[1].tolist()
+            == bogus_samples.destinations.dropna().values_host.tolist()
+        )
 
     assert num_samples == 100
 
 
 @pytest.mark.skipif(isinstance(torch, MissingModule), reason="torch not available")
-def test_cugraph_loader_from_disk_subset_renumbered():
+def test_cugraph_loader_e2e_coo():
+    m = [2, 9, 99, 82, 9, 3, 18, 1, 12]
```

Comment:
I really like this test and we should add this for cugraph-dgl too.

Reply:
Yeah, this test caught several bugs immediately after I added it.